src/test/java/com/rabbitmq/client/test/TestUtils.java

```diff
@@ -31,4 +34,14 @@ public static ConnectionFactory connectionFactory() {
         return connectionFactory;
     }
 
+    public static void close(Connection connection) {
+        if(connection != null) {
+            try {
+                connection.close();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
If we are simply ensuring that a connection is closed, does it make sense to throw a `RuntimeException` or should it be ignored? In the latter case we could rename the method to `ensureClosed`.